test/gpu/native/diags.good

```diff
@@ -6,7 +6,7 @@ Start
 0 (cpu): diags.chpl:10: allocate 48B of _EndCount(AtomicT(int(64)),int(64)) at 0xPREDIFFED
 0 (cpu): diags.chpl:10: free at 0xPREDIFFED
 0 (cpu): diags.chpl:12: allocate 168B of [domain(1,int(64),false)] locale at 0xPREDIFFED
-0 (cpu): diags.chpl:12: allocate 16B of array elements at 0xPREDIFFED
+0 (cpu): diags.chpl:12: allocate 128B of array elements at 0xPREDIFFED
```

Comment:
Could this be from your locale model changes? This line is where we do `here.getChild` vs `here.gpus`. Apparently we were already doing some allocation there already, but now we are making a bigger allocation?

I wonder if we run this manually with `--devel` this line would change to point to a more precise location in the modules where this allocation is coming from.